src/main/resources/db/schema/SL-1876-multi-hearing-db-changes.yaml

```diff
@@ -0,0 +1,170 @@
+databaseChangeLog:
```

Comment:
As we discussed yesterday we need  to add a column to hearing_part which points the hearing entity.